caikit/runtime/service_generation/data_stream_source.py

```diff
@@ -200,10 +200,10 @@ def _load_from_file_without_extension(cls, fname) -> DataStream:
         full_fname = cls._get_resolved_source_path(fname)
         log.debug3("Attempting to guess file type for file: %s", full_fname)
         for factory_method in (
+            DataStream.from_multipart_file,
```

Comment:
Can we add a note here explaining why multipart_file needs to be first?

Reply:
docstring updated!